rest/data/commit_queue.go

```diff
@@ -342,6 +342,10 @@ func getAndEnqueueCommitQueueItemForPR(ctx context.Context, env evergreen.Enviro
 		return nil, pr, errors.Wrapf(errNoCommitQueueForBranch, "repo '%s:%s', branch '%s'", info.Owner, info.Repo, baseBranch)
 	}
 
+	if projectRef.CommitQueue.MergeQueue == model.MergeQueueGitHub {
+		return nil, pr, errors.Wrapf(errors.New("project is using GitHub merge queue"), "repo '%s:%s', branch '%s'", info.Owner, info.Repo, baseBranch)
+	}
+
```

Comment:
Is there any way for the "Add to commit queue" button  on the patch page to still work for for projects that use the github merge queue (I see this is called by the EnqueuePatch resolver)? 
If not, do we have a ticket that will hide that button when appropriate?

Reply:
@minnakt mentioned this as well. 

I think users can only add to the GitHub merge queue from the GitHub UI. Disabling the commit queue doesn't hide that button today, so I don't think it's a regression to leave it. I have a note to myself to either open a ticket to hide this or to defer this to later, when/if we remove Evergreen's native commit queue.